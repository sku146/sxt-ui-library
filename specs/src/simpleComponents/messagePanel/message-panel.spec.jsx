import React from 'react';
import renderer from 'react-test-renderer';
import MessagePanel from 'src/simpleComponents/messagePanel/message-panel';

describe("Message Panel", () => {

  let defaultProps = {
    name: 'testMessagePanel',
    msgType: 'error', // success, error, info, warning, plain(with white background), transparent
    pointPosition: 'left',
    pointing: true,
  };

  const RendererComponent = (props) => {
    return renderer.create(<MessagePanel {...props}><div>test</div></MessagePanel>);
  };

  it('component should render as expected', ()=> {
    const component = RendererComponent(defaultProps);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
