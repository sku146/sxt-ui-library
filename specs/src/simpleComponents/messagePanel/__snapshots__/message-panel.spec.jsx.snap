// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Panel component should render as expected 1`] = `
<div
  className="ui pointing basic label message-error position-left"
  data-bdd-selector="testMessagePanelErrorMsg"
  onClick={[Function]}
>
  <div>
    test
  </div>
</div>
`;
